examples/hackweek_demo/run_demo.py

```diff
@@ -16,13 +16,13 @@
 
 # pylint: disable=invalid-name
 
-st.set_page_config(page_title="Trading Competition Dashboard", layout="centered")
+st.set_page_config(page_title="Trading Competition Dashboard", layout="wide")
```

Comment:
based

Reply:
Oops. More columns in ticker means it looks better in wide mode

Reply:
everything looks better in wide mode 👌